README.md

```diff
@@ -27,6 +27,10 @@ This action uses the [REUSE helper tool](https://github.com/fsfe/reuse-tool). Fo
 You can include the following lines in your workflow .yml file to run the lint subcommand:
 
 ```yml
+# SPDX-FileCopyrightText: 2020 Free Software Foundation Europe e.V. <https://fsfe.org>
```

Comment:
Could you please make this 2022? Let's take the current year, although the snippet was created earlier.